.buildkite/scripts/gcp_auth.ps1

```diff
@@ -0,0 +1,20 @@
+Write-Host "~~~ Authenticating GCP"
+# Secrets must be redacted
+# https://buildkite.com/docs/pipelines/managing-log-output#redacted-environment-variables
```

Comment:
For the record, secrets are not redacted here; that can only be done in the `pre-command` hook, unfortunately.

Regardless, I'm just working on using OIDC so we won't need this approach anymore